aspnetcore/fundamentals/logging/index.md

```diff
@@ -125,6 +125,74 @@ To write logs in the `Program` class of an ASP.NET Core app, get an `ILogger` in
 
 [!code-csharp[](index/samples/3.x/TodoApiSample/Program.cs?name=snippet_LogFromMain&highlight=9,10)]
 
+Logging in `ConfigureServices` isn't directly supported. However, a static third-party logger can be used. In the following example, a [Serilog](https://serilog.net/) logger is used to log in `CreateHostBuilder`:
```

Comment:
```suggestion
Logging in `ConfigureServices` isn't directly supported. However, a separate logger can be used. In the following example, a [Serilog](https://serilog.net/) logger is used to log in `CreateHostBuilder`:
```

Reply:
This description says you can't log in ConfigureServices, but the sample shows how to log in CreateHostBuilder? The sample should also log in ConfigureServices?

Reply:
Maybe better as “logging while building the host” isn’t supported. Is that accurate?

Reply:
Like that? ☝️